hedera-node/hedera-app/src/main/java/com/hedera/node/app/workflows/SolvencyPreCheck.java

```diff
@@ -128,13 +133,21 @@ public void checkSolvency(
             throw new InsufficientNetworkFeeException(INSUFFICIENT_TX_FEE, totalFee);
         }
         if (availableBalance < fees.networkFee()) {
-            throw new InsufficientNetworkFeeException(INSUFFICIENT_PAYER_BALANCE, totalFee);
+            if (ingestCheck) { // throw different exception for ingest
+                throw new InsufficientBalanceException(INSUFFICIENT_PAYER_BALANCE, totalFee);
+            } else {
+                throw new InsufficientNonFeeDebitsException(INSUFFICIENT_ACCOUNT_BALANCE, totalFee);
```

Comment:
Yick!  Definitely want to remove this false exception in the future.  Labeling insufficient funds to pay a _fee_ as `InsufficientNonFeeDebits` is just bad...

Reply:
Updated with the change suggested.